src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.TryGet.cs

```diff
@@ -65,6 +65,126 @@ public bool GetBooleanValue()
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if trying to get the value of JSON token that is not a <see cref="JsonTokenType.Number"/>.
+        /// <seealso cref="TokenType" />
+        /// </exception>
+        /// <exception cref="FormatException">
+        /// Thrown if the value of JSON token is either too large or too small for a <see cref="float"/>.
+        /// <seealso cref="TokenType" />
+        /// </exception>
+        public float GetSingle()
+        {
+            float value;
+            if (!TryGetSingleValue(out value))
+            {
+                throw ThrowHelper.GetFormatException_Single();
+            }
+            return value;
+        }
+
+        /// <summary>
+        /// Reads the next JSON token value from the source and parses it to a <see cref="double"/>.
+        /// Returns the value if the entire UTF-8 encoded token value can be successfully parsed to a <see cref="double"/>
+        /// value.
+        /// Throws exceptions otherwise.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if trying to get the value of JSON token that is not a <see cref="JsonTokenType.Number"/>.
+        /// <seealso cref="TokenType" />
+        /// </exception>
+        /// <exception cref="FormatException">
+        /// Thrown if the value of JSON token is either too large or too small for a <see cref="double"/>.
+        /// <seealso cref="TokenType" />
+        /// </exception>
+        public double GetDouble()
+        {
+            double value;
+            if (!TryGetDoubleValue(out value))
+            {
+                throw ThrowHelper.GetFormatException_Double();
+            }
+            return value;
+        }
+
+        /// <summary>
+        /// Reads the next JSON token value from the source and parses it to a <see cref="decimal"/>.
+        /// Returns the value if the entire UTF-8 encoded token value can be successfully parsed to a <see cref="decimal"/>
+        /// value.
+        /// Throws exceptions otherwise.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
+        /// Thrown if trying to get the value of JSON token that is not a <see cref="JsonTokenType.Number"/>.
+        /// <seealso cref="TokenType" />
+        /// </exception>
+        /// <exception cref="FormatException">
+        /// Thrown if the value of JSON token is either too large or too small for a <see cref="decimal"/>.
+        /// <seealso cref="TokenType" />
+        /// </exception>
+        public decimal GetDecimal()
+        {
+            decimal value;
+            if (!TryGetDecimalValue(out value))
+            {
+                throw ThrowHelper.GetFormatException_Decimal();
+            }
+            return value;
+        }
+
         /// <summary>
         /// Reads the next JSON token value from the source and parses it to a <see cref="int"/>.
```

Comment:
Can you please fix the `a <see cref="int"/>` to `an a <see cref="int"/>` like in the other APIs while you are updating the code here.